build_tools/python/benchmark_suites/iree/x86_64_benchmarks.py

```diff
@@ -37,9 +37,11 @@ class Linux_x86_64_Benchmarks(object):
       ])
 
   def _generate(
-      self, model_group_and_threads: List[common_definitions.Model],
+      self,
+      model_group_and_threads: List[common_definitions.Model],
```

Comment:
The type should be `List[Tuple[common_definitions.Model, List[int]]]`?

In general, I think we might want to define a dataclass for "model group and threads".

Reply:
I created a new data class `CpuBenchmarkConfig` in common_definitions.